.makim.yaml

```
@@ -13,6 +13,7 @@ groups:
           rm -fr .pytest_cache
           rm -fr .ruff_cache
           rm -fr .mypy_cache
+          rm ./tests/smoke/makim_log.txt
```

```suggestion
          rm -f ./tests/smoke/makim_log.txt
```
`-f` prevents to fail if the file doesn't exist

---

> `-f` prevents to fail if the file doesn't exist


that makes sense, I've made the change, thanks for the suggestion!